Objects/typeobject.c

```diff
@@ -252,7 +252,7 @@ _PyType_InitCache(PyInterpreterState *interp)
 
 
 static unsigned int
-_PyType_ClearCache(struct type_cache *cache)
+_PyType_ClearCache(PyInterpreterState *interp, struct type_cache *cache)
```

Comment:
Hum, the cache parameter sounds redundant. Please remove it and get the cache with:

`struct type_cache *cache = &interp->type_cache;`

So you can remove `struct type_cache *cache = get_type_cache();` in PyType_ClearCache().

Reply:
Thanks vstinner, good suggestion.